WordPress/openverse-frontend
# Add meta tags to pages with `new_header` enabled

## Fixes


Fixes #1826 by @zackkrida

## Description


This PR adds `<meta name="robots" content="all">` (when the `new_header` is enabled) to the following pages:
- search results (all content/image/audio)
- home page
- content pages (about/search help, etc.)

## Testing Instructions

Run the app and check that without `new_header`, the `robots` tag is set to 'noindex', and with the new header, the pages in the list have `robots` set to `all`. You can check this by entering `document.querySelector('meta[name="robots"]').content` into the devtools console.

Current values on the staging homepage:
<img width="415" alt="Screen Shot 2022-10-28 at 5 52 27 PM" src="https://user-images.githubusercontent.com/15233243/198660589-100df13e-e6bd-48e9-b973-fbeb5bec5bfd.png">

Values from this PR's homepage:
<img width="412" alt="Screen Shot 2022-10-28 at 5 53 18 PM" src="https://user-images.githubusercontent.com/15233243/198660730-b25b7fe3-b43a-4c99-834b-58616cdeb972.png">

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
